<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win_main</class>
 <widget class="QWidget" name="win_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fibonacci Client</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>none</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_input">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_length">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>none</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_length">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer length:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Input:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_serverip">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_serverip">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Server IP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_timeout">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>20000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_timeout">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Timeout:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
